src/api/controller/MyController.hpp

```diff
@@ -112,32 +99,31 @@ class MyController : public oatpp::web::server::api::ApiController {
 
     auto responseDto = bakcyl::api::RegisterResponseDto::createShared();
 
-    if (json && json->username && json->password && json->email) {
-      std::pair<int, std::string> registerPair =
-          auth.registerUser(json->username, json->password, json->email);
-
-      int errorCode = registerPair.first;
-      std::string token = registerPair.second;
-
-      if (errorCode == 0)
-        responseDto->success = true;
-      else
-        responseDto->success = false;
-      responseDto->errorCode = errorCode;
-
-      if (errorCode == 0) {
-        auto response = createDtoResponse(Status::CODE_201, responseDto);
-        response->putHeader("Authorization", "Bearer " + token);
-        return response;
-      } else if (errorCode == 1) {
-        return createDtoResponse(Status::CODE_400, responseDto);
-      } else {
-        return createDtoResponse(Status::CODE_409, responseDto);
-      }
-    } else {
+    if (!json && !json->username && !json->password && !json->email) {
       responseDto->success = false;
       return createDtoResponse(Status::CODE_400, responseDto);
     }
+
+    std::pair<int, std::string> registerPair =
+        auth.registerUser(json->username, json->password, json->email);
```

Comment:
const auto& [errorCode, token] =  auth.registerUser(json->username, json->password, json->email);

Reply:
the commit 1e8d812 solved the issue on #90